test/Microsoft.IdentityModel.Tests/ReferenceSaml.cs

```diff
@@ -0,0 +1,87 @@
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.ObjectModel;
+using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens.Saml;
+
+namespace Microsoft.IdentityModel.Tests
+{
+    public class ReferenceSaml
+    {
+        public static SamlAssertion SamlAssertion
+        {
+            get
+            {
+                return new SamlAssertion(Default.SamlAssertionID, Default.Issuer, DefaultSamlData.IssueInstant, SamlConditions, null, new Collection<SamlStatement>{ SamlAttributeStatement }, null);
+            }
+        }
+
+        public static SamlAttributeStatement SamlAttributeStatement
+        {
+            get
+            {
+                var attribute1 = new SamlAttribute(Default.NameSpace, DefaultClaimType.Country.ToString(), ClaimSets.DefaultClaimsMapping[DefaultClaimType.Country]);
+                var attribute2 = new SamlAttribute(Default.NameSpace, DefaultClaimType.Email.ToString(), ClaimSets.DefaultClaimsMapping[DefaultClaimType.Email]);
+                var attribute3 = new SamlAttribute(Default.NameSpace, DefaultClaimType.GivenName.ToString(), ClaimSets.DefaultClaimsMapping[DefaultClaimType.GivenName]);
+                var attribute4 = new SamlAttribute(Default.NameSpace, DefaultClaimType.HomePhone.ToString(), ClaimSets.DefaultClaimsMapping[DefaultClaimType.HomePhone]);
+
+                // Add attribute: AttributeName = "role", value = "Developer", "Sales", "role1", "roles1"
+                var attribute5 = new SamlAttribute(Default.NameSpace, DefaultClaimType.Role.ToString(), new Collection<string> { DefaultRoles.Developer.ToString(), DefaultRoles.Sales.ToString(), DefaultRoles.role1.ToString(), DefaultRoles.roles1.ToString() });
+
+                // Add attribute: AttributeName = "name", value = "Jean-S�bastien"
+                var attribute6 = new SamlAttribute(Default.NameSpace, DefaultClaimType.Name.ToString(), ClaimSets.DefaultClaimsMapping[DefaultClaimType.Name]);
+
+                var attributes = new Collection<SamlAttribute> { attribute1, attribute2, attribute3, attribute4, attribute5, attribute6 };
+                return new SamlAttributeStatement(SamlSubject, attributes);
+            }
+        }
+
+        public static SamlConditions SamlConditions
+        {
+            get
+            {
+                var audiences = new Collection<string> { Default.Audience };
+                var conditions = new Collection<SamlCondition> { new SamlAudienceRestriction(audiences) };
+                return new SamlConditions(DefaultSamlData.NotBefore, DefaultSamlData.NotOnOrAfter, conditions);
+            }
+        }
+
+        public static SamlSubject SamlSubject
+        {
+            get { return new SamlSubject(); }
+        }
+    }
+
+    public class DefaultSamlData
```

Comment:
Please add this these values to the Default class, don't add another place to look for defaults.